content/300-guides/300-upgrade-guides/200-upgrading-versions/050-upgrading-to-prisma-3/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Upgrade to Prisma 3'
+metaTitle: 'Upgrade to Prisma 3'
+metaDescription: 'Guides on how to upgrade to Prisma 3'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+To upgrade from version 2.x to 3.x, you need to update both the `prisma` and `@prisma/client` packages. Both the `prisma` and `@prisma/client` packages install with a caret `^` in their version number to safe guard against breaking changes.
+
+To ignore the caret `^` and upgrade across major versions, you can use the `@3` tag when upgrading with `npm`, or `yarn` .
+
+<TabbedContent tabs={[<FileWithIcon text="npm" icon="file"/>, <FileWithIcon text="yarn" icon="file"/>]}>
+
+<tab>
+
+```terminal
+npm install prisma@3 @prisma/client@3
+```
+
+</tab>
+
+<tab>
+
+```terminal
+yarn upgrade prisma@3 @prisma/client@3
+```
+
+</tab>
+
+</TabbedContent>
+
+<Admonition type="alert">
+
+Before upgrading, check each of the below **breaking changes** to see how the upgrade might affect your application.
+
+</Admonition>
+
+</TopBlock>
+
+## Breaking changes
+
+There are a number of breaking changes to be aware of when upgrading from version 2.x to 3.x.
+
+### [Referential actions](../../../../concepts/components/prisma-schema/relations/referential-actions)
+
+The introduction of referential actions in version 3.x removes the safety net in the Prisma Client that had previously prevented cascading deletes at runtime.
+
+A potential loss of data may occur when you upgrade depending on which workflow ([Introspection](../../../../concepts/components/introspection) or [Migration](../../../../concepts/components/prisma-migrate)) your application uses.
+
+See [Referential action upgrade paths](./referential-actions) for more information.
+
+### [$queryRaw](../../../../concepts/components/prisma-client/raw-database-access)
+
+From version 3.x onwards, the `$queryRaw` method now only supports a template literal.
+
+New `$queryRawUnsafe` and `$executeRawUnsafe` methods have been added which accept a string or template string. These methods allow you to pass a raw template string to the database.
```

Comment:
```suggestion
This means that if your application relied on `$queryRaw` calls using *strings*, those calls will not work anymore. We advise you to use template literals wherever possible for security reasons or resort to `$queryRawUnsafe` otherwise, after carefully escaping queries to prevent SQL injections.

You can learn more about the new `$queryRaw` and `$queryRawUnsafe` methods in the [Raw database access](../../../../concepts/components/prisma-client/raw-database-access) section of the docs.
```